README.md

```diff
@@ -1 +1,67 @@
+| Mescolato                              | Pipo            | clear your mind                                                                                                                                                                                                                                                                                                                                                                          | https://signorpipo.itch.io/mescolato                     | Yes         | https://github.com/signorpipo/mescolato        |
+| De-Evil-ery                            | Elia & Marco    | That stupid Prince Ugo! He’s trying to wake up the Princess after you finally trapped her in the mystic blue sphere! Don’t panic! De-Evil-ery is the company that can support you in your fights! Destroy the mecha-birds sent by Prince Ugo to wake up the Princess, and gain evilness points to buy new weapons! Be Careful though! If the true love meter fills up… IT’S GAME OVER!!! | https://signorpipo.itch.io/de-evil-ery                   | Yes         | https://github.com/signorpipo/devilery         |
+| Sketchy                                | Pipo & Yinch    | Have you ever wanted to test if the new sofa will have enough place in the living room? Well, Sketchy is here to help you! Create, grab, scale, rotate, move, whatever! You can also share the room so that your best friend can approve the new arrangement Well, some use this to create whatever they want so, you can do that too!                                                   | https://signorpipo.itch.io/sketchy                       | Yes         | https://github.com/SignorPipo/sketchy          |
+| PPlayground                            | Pipo            | Playground template project for the Wonderland Engine                                                                                                                                                                                                                                                                                                                                    | https://signorpipo.itch.io/pplayground-wonderland-engine | Yes         | https://github.com/signorpipo/wle-pplaygrounds |
+| Labyroots                              | I Polaretti     | The mother of all trees... After landing on earth, her spores began to spread, while the roots protected her heart. Many fought to defeat her, and to save our land, but they all turned into... It's time for you to venture into her maze, and put this madness to an end. The mother of all trees must succumb.                                                                       | https://heyvr.io/arcade/games/labyroots                  | Yes         | https://github.com/signorpipo/labyroots        |
```

Comment:
```suggestion
| Labyroots                              | Pipo & ??     | The mother of all trees... After landing on earth, her spores began to spread, while the roots protected her heart. Many fought to defeat her, and to save our land, but they all turned into... It's time for you to venture into her maze, and put this madness to an end. The mother of all trees must succumb.                                                                       | https://heyvr.io/arcade/games/labyroots                  | Yes         | https://github.com/signorpipo/labyroots        |
```

I guess their team name was I Polaretti, but maybe list the others of the team instead?